README.md

```diff
@@ -1,8 +1,48 @@
 
+I have the ability to break things down and communicate to any level. I have the right mix of strategic thinking and the ability to influence gained over the years working as an sole propritor. I am extremely good at building up and nurturing interpersonal relationships. I have a high awareness of business goals and business organization. I am very attentive to details and am always willing to take the extra step to get things accomplished. I also have utilized requirement elicitation skills in the previous job to understand all the client's needs and objectives. I am very good at making sure things get done and get done right. I have fine people skills, am analytical, and can think on my feet combined with savvy, computer skills. I have utilized the skills needed for presenting on several occasions academically and professionally . As an investor in Forex and Stocks, I have the basic knowledge of how wealth management works. Because of my graduate degree in Electronics and Communications Engineering from the University of Kent, England I am well versed in a good range of programming and database skills across many platforms. Therefore, I wish to be considered for a Business System Analyst role where I can use my skills and work in a team setting so I can also work on my leadership skills.
+
+## Skills
+
+**Business Skills:** Requirement Elicitation, Presentation, Documentation, Effective Communication and interpersonal, Decision-Making Skills, Networking, Data Analysis Skills, Problem-Solving
+
+**Technical Skills:** RDBMS/SQL queries, Data Modeling/Mapping, Data Analytics using Python, Pandas, NumPy., Agile/Scrum, Linux/Bash Scripts, GitHub, MS Excel(Pivot table/Formulae), MS Visio, MS Word, MS PowerPoint, HTML5 and CSS, Google Cloud Platform, Matlab, C Pragramming, Salesforce
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_awaiszafar](https://github.com/jarviscanada/jarvis_data_eng_awaiszafar)
+
+
+**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_awaiszafar/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_AwaisZafar/tree/feature/creating-ddl/linux_sql/scripts)]: Linux Cluster Monitoring app is used by the Linux cluster agents to monitor the hardware specifications and memory usage information in real time and over one minute intervals. A System Required Document (SRD) was prepared by me initially containing all the project objectives and identified the stakeholders. Performed requirement elicitation technique through open interviews with product developers. Data models were created using E-R Diagrams. To collect the data RDBMS/SQL queries were written along with linux bash scripts and crontab was also to save the data over minute intervals. Lastly, after completion of the project it was presentated to product developer with documentation prepared containing all the technical aspects of the project and was deployed using GitHub.
```

Comment:
Developed a Linux cluster agent to monitor the hardware specifications and memory usage information in real-time. A System Required Document (SRD) was prepared by me initially, containing all the project objectives and identifying the stakeholders. Performed requirement elicitation technique through open interviews with product developers. Data models were created using E-R Diagrams. To collect the data, RDBMS/SQL queries were written along with Linux bash scripts, and crontab was also to save the data over minute intervals. Lastly, after completion of the project, it was presented to the product developer with documentation prepared containing all the technical aspects of the project and was deployed using GitHub.